.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "haskell.toolchain": {
+    "cabal": "3.10.3.0",
+    "hls": "2.8.0.0",
+    "stack": "2.15.7"
```

Comment:
I wonder if VS Code or the extension will complain if we put the other version pins here, so they’re all in one place.

Or is that a bad idea because VS Code users might think that it’s _actually_ pinning Ormolu, etc.